rasa/shared/utils/schemas/model_config.yml

```diff
@@ -0,0 +1,33 @@
+allowempty: True
+mapping:
+  version:
+    type: "str"
+    required: False
+    allowempty: False
+  language:
+    type: "str"
+    required: True
```

Comment:
double check if this is really required

Reply:
`language` [should not be required](https://rasa-hq.slack.com/archives/C36SS4N8M/p1613480592285800)